code/modules/mob/living/silicon/robot/modules/module_research.dm

```diff
@@ -1,6 +1,7 @@
 /obj/item/robot_module/research
 	name = "research module"
 	display_name = "Research"
+	module_category = ROBOT_MODULE_DISABLED
```

Comment:
these modules are 'useless' because no one really plays them and they're just cluttering up space in the module selection menu. Who actually picks the standard module? It's useless, you have NOTHING to do with it.

Reply:
and your justification that "no one really plays them" is backed up by what? A pole? Asking around? We cant just remove things because we individually might not see people playing or think its unused because there just might be people who play it that you haven't met and taking that functionality away can be problematic. Now I know you don't have access to dev poles, but if you want I can make a pole so that we are certain.

Reply:
all you need to do is ask, ill put one up now

Reply:
👍

Reply:
poll shows research module should be kept

Reply:
> poll shows research module should be kept

it has now been kept